libs/platform/experience/src/services/layout/plugins/layout.plugin.ts

```diff
@@ -9,10 +9,38 @@ import {
 } from '../../../models';
 import { LayoutStyles, LayoutStylesOptions } from '../layout.model';
 
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  export interface Layouts {}
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  export interface LayoutProperty {}
+
+  export interface LayoutStylesProperties {
+    layout?: LayoutStylesOptions | LayoutTypes;
+
+    // @deprecated since 1.2 will be removed.
```

Comment:
```suggestion
    // @deprecated since 1.2 will be removed, use the ... plugin instead
```

also in other notes below